### DISCUSSION TITLE
Unhandled Exception occurred

### DISCUSSION THREAD

**Discussion by Presyon** [fraxiinus/ReplayBook#324] [Troubleshooting]
When I add the league of legends exe it looks just fine but after I exit settings and it updates, the attached picture shows and it exits. I couldnt find anything like this in the issues or discussions. I have been using this program to look at replays and its great but now trying to play a replay from a previous patch is rough. Now that I have been trying to get a previous patch to work, the up to date replays dont work,
![Screenshot 2024-11-08 140158](https://github.com/user-attachments/assets/9e4bff6a-8763-4872-aae7-4577dfcbf9c5)
![Screenshot 2024-11-08 135641](https://github.com/user-attachments/assets/def84dd0-9389-4818-bfe6-c42580a0d9ba)

**Comment by Presyon** [comment]
Fixed it by running as administrator. I'm an idiot.

**Reply by fraxiinus** [reply]
Don't put the application in the root C drive, that is a protected directory. Simply put ReplayBook somewhere like your Documents or similar location.

**Reply by Presyon** [reply]
It was in Documents before and doing the same thing. I moved it to C: to see if anything changed, thats why I didnt move it back.
